docs/notebooks/thomson.ipynb

```diff
@@ -36,7 +36,7 @@
   },
```

Comment:
<p>Outputs are back, which is completely okay in development, let's just make sure they're cleared before we merge it (I keep forgetting about it, so I'll be adding some additional CI checks for that later).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Oops, yes I'll try to remember to clear them. </p><p> </p><p>I wouldn't make it a required check (maybe a warning): there are some cases like computationally intensive code (thinking of the proton radiography module) where leaving some output is desirable? </p>

Reply:
Yes, of course, which is why my intention is to have a label like we have for "no changelog entry needed" to allow those :)